utils/getFormatsDescription.test.ts

```diff
@@ -0,0 +1,176 @@
+import { expect, test } from "vitest";
+import getFormatsDescription from "./getFormatsDescription";
+
+// formats.length=1, qty=1, text, descriptions.length=1
+test("Render album description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "1",
+        text: "Pink Marble",
+        descriptions: ["LP"],
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl LP"));
+
+// formats.length=1, qty>1, text, descriptions.length=1
+test("Render album description", () =>
```

Comment:
also, these test names should be unique and identify what case this is handling (at a high level, without implementation details)

```suggestion
test("can generate a description where a single item has multiple quantity", () =>
```